[2023-12-30T15:21:43.150+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: run_dbt_model_cosmos.dbt_dwh_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T15:21:43.160+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: run_dbt_model_cosmos.dbt_dwh_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T15:21:43.160+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T15:21:43.160+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T15:21:43.160+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T15:21:43.173+0000] {taskinstance.py:1383} INFO - Executing <Task(DbtTestLocalOperator): dbt_dwh_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test> on 2022-09-10 00:00:00+00:00
[2023-12-30T15:21:43.178+0000] {standard_task_runner.py:55} INFO - Started process 19603 to run task
[2023-12-30T15:21:43.181+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'run_dbt_model_cosmos', 'dbt_dwh_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models_cosmos.py', '--cfg-path', '/tmp/tmpyf6f_qt3']
[2023-12-30T15:21:43.181+0000] {standard_task_runner.py:83} INFO - Job 182: Subtask dbt_dwh_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test
[2023-12-30T15:21:43.232+0000] {task_command.py:376} INFO - Running <TaskInstance: run_dbt_model_cosmos.dbt_dwh_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T15:21:43.291+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=run_dbt_model_cosmos
AIRFLOW_CTX_TASK_ID=dbt_dwh_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T15:21:43.298+0000] {base.py:71} INFO - Using connection ID 'postgres_default' for task execution.
[2023-12-30T15:21:43.299+0000] {local.py:136} INFO - Cloning project to writable temp directory /tmp/tmpoyovu_gk from //opt/airflow/dags/../dbt/dwh
[2023-12-30T15:21:43.372+0000] {local.py:157} INFO - Trying to run the command:
 ['dbt', 'test', '--vars', '{"target_date": "2022-09-10"}', '--models', 'raw_daily_customer_drivers', '--profile', 'postgres_profile']
From /tmp/tmpoyovu_gk/dbt_project
[2023-12-30T15:21:43.372+0000] {adapted_subprocesshook.py:53} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T15:21:43.372+0000] {adapted_subprocesshook.py:66} INFO - Running command: ['dbt', 'test', '--vars', '{"target_date": "2022-09-10"}', '--models', 'raw_daily_customer_drivers', '--profile', 'postgres_profile']
[2023-12-30T15:21:43.382+0000] {adapted_subprocesshook.py:77} INFO - Output:
[2023-12-30T15:21:45.309+0000] {adapted_subprocesshook.py:87} INFO - [0m15:21:45  Running with dbt=1.4.6
[2023-12-30T15:21:45.369+0000] {adapted_subprocesshook.py:87} INFO - [0m15:21:45  Unable to do partial parsing because config vars, config profile, or config target have changed
[2023-12-30T15:21:45.370+0000] {adapted_subprocesshook.py:87} INFO - [0m15:21:45  Unable to do partial parsing because profile has changed
[2023-12-30T15:21:45.370+0000] {adapted_subprocesshook.py:87} INFO - [0m15:21:45  Unable to do partial parsing because env vars used in profiles.yml have changed
[2023-12-30T15:21:47.064+0000] {adapted_subprocesshook.py:87} INFO - [0m15:21:47  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T15:21:47.065+0000] {adapted_subprocesshook.py:87} INFO - [0m15:21:47
[2023-12-30T15:21:47.066+0000] {adapted_subprocesshook.py:87} INFO - [0m15:21:47  Nothing to do. Try checking your model configs and model specification args
[2023-12-30T15:21:48.433+0000] {adapted_subprocesshook.py:91} INFO - Command exited with return code 0
[2023-12-30T15:21:48.475+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/baseoperator.py:1390: RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
  context['ti'].xcom_push(key=key, value=value, execution_date=execution_date)

[2023-12-30T15:21:48.493+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=run_dbt_model_cosmos, task_id=dbt_***_project.raw_daily_customer_drivers.raw_daily_customer_drivers_test, execution_date=20220910T000000, start_date=20231230T152143, end_date=20231230T152148
[2023-12-30T15:21:48.569+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-30T15:21:48.607+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
